backend/app.py

```diff
@@ -87,6 +88,21 @@ def get_all_categories():
     return jsonify(response)
 
 
+@cross_origin
+@app.route("/api/recommendation_engine", methods=["GET"])
+def get_recommendation():
+    print("GET /api/recommendation_engine")
```

Comment:
Not a change request but comment for further work: frontend/src/pages/ResultsPage/SearchResults.tsx routes to /api/job/results/${searchValues.numberOfSearchResults}. Will need to tie the recommendation_engine and results/<int:number> route